server/vulnerabilities/nvd/cpe.go

```diff
@@ -462,3 +467,12 @@ func TranslateSoftwareToCPE(
 
 	return nil
 }
+
+func containsNonASCII(s string) bool {
```

Comment:
Can you please add a test for this function?
Preferably both true and false returned.

Reply:
```
	expectTrue := containsNonASCII("Девушка Фонарём")
	assert.True(t, expectTrue)
	expectFalse := containsNonASCII("Only Ascii 7658765^&*%$^*%&$")
	assert.False(t, expectFalse)
```